test/config/integration/server_xds.lds.yaml

```diff
@@ -0,0 +1,19 @@
+versionInfo: "0"
+resources:
+- "@type": type.googleapis.com/envoy.api.v2.Listener
+  name: listener_0
+  address:
+    socketAddress:
+      address: {{ ntop_ip_loopback_address }}
+      portValue: 0
+  filterChains:
+  - filters:
+    - name: http_connection_manager
+      config:
+        codecType: HTTP2
+        drainTimeout: 5s
+        statPrefix: router
+        rds:
+          routeConfigName: route_config_0
+          configSource: { path: {{ rds_json_path }} }
+        httpFilters: [{ name: router, config: { deprecatedV1: true }}]
```

Comment:
Not sure what the "from json to yaml" is referring to, can you clarify?

On the `@type` annotations, these are a product of the generic handling of subscription types and ADS multiplexing support. I don't expect we'll see this stuff in hand-written configs in v2, instead, we will allow the bootstrap to specify both listeners and clusters, like v1, so this should only show up in test files like the above, or where folks want filesystem based dynamic config. I agree it's a bit ugly, but hand-written configs are really only support in a best effort manner, one of the v2 API principles is that the API is primarily for machines.